android/src/main/java/io/didomi/reactnative/DidomiModule.kt

```diff
@@ -1132,6 +1182,58 @@ class DidomiModule(reactContext: ReactApplicationContext) : ReactContextBaseJava
         return Array(readableArray.size()) { i -> readableArray.getString(i) }
     }
 
+    private fun buildDidomiInitializeParameters(jsonParameters: String): DidomiInitializeParameters {
+        val jsonObject = JSONObject(jsonParameters)
+        
+        return DidomiInitializeParameters(
+            apiKey = jsonObject.getString("apiKey"),
+            localConfigurationPath = jsonObject.opt("localConfigurationPath")?.toString(),
+            remoteConfigurationUrl = jsonObject.opt("remoteConfigurationUrl")?.toString(),
+            providerId = jsonObject.opt("providerId")?.toString(),
+            disableDidomiRemoteConfig = (jsonObject.opt("disableDidomiRemoteConfig") as? Boolean) ?: false,
+            languageCode = jsonObject.opt("languageCode")?.toString(),
+            noticeId = jsonObject.opt("noticeId")?.toString(),
+            tvNoticeId = jsonObject.opt("androidTvNoticeId")?.toString(),
+            androidTvEnabled = (jsonObject.opt("androidTvEnabled") as? Boolean) ?: false,
+            countryCode = jsonObject.opt("countryCode")?.toString(),
+            regionCode = jsonObject.opt("regionCode")?.toString(),
+            isUnderage = (jsonObject.opt("isUnderage") as? Boolean) ?: false
```

Comment:
I i remember correctly, when `isUnderage` value is not set, it should stay `null`?

Reply:
Or is it just for the `setUser` method

Reply:
yes,  `isUnderage` is not nullable/optional in `DidomiInitializeParameters` while in `DidomiUserParameters` it is.